app/controllers/api/v1x1/portfolio_items_controller.rb

```diff
@@ -13,10 +13,16 @@ def show
 
       def render_item(item)
```

Comment:
How about changing it to `def render_item(item, is_deleted = false)` and determine `is_deleted` in `show` method?